### PR TITLE
Weekly meeting minutes 2017-12-14

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2017-12-14-startup.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        at the workshop).
+    -   Sorting out room availability, but probably no showstopper,
+        either on Monday morning (will start at 10 am but
+        registration will open before that) or on Thursday (the plan is
+        that the 150 seat room will be extended by Vidyo in a smaller
+        room if needed).
+    -   Liz: Hotel information is very important for US Lab people.
+        Fallback would be for us to gather information ourselves and
+        pass it on in the announcement next week.
+    -   First round at identifying conveners was done.
```
Comment (hegner) [inline organization/_posts/2017-12-14-startup.md]: not true. we listed names, but didn't contact any of them. Doesn't really qualify for *identifying*

File: organization/_posts/2017-12-14-startup.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+Licensing
+---------
+
+-   ATLAS have been considering their licensing situation - discussion at ICB this week.
+
+Training
+--------
+
+-   NTR
```
Comment (hegner) [inline organization/_posts/2017-12-14-startup.md]: nothing to report

File: organization/_posts/2017-12-14-startup.md (hunk 1 above)
Comment (graeme-a-stewart) [reply organization/_posts/2017-12-14-startup.md]: Fixed

File: organization/_posts/2017-12-14-startup.md (hunk 2 above)
Comment (graeme-a-stewart) [reply organization/_posts/2017-12-14-startup.md]: Fixed

**Pull request merged**